packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -334,8 +334,17 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
 
   /// State for all pointers which are currently down.
   ///
-  /// The state of hovering pointers is not tracked because that would require
-  /// hit-testing on every frame.
+  /// This map caches the hit test result done when the pointer goes down
+  /// ([PointerDownEvent] and [PointerPanZoomStartEvent]). This hit test result
+  /// will be used throughout the entire tap; that is, the pointer is seen as
```

Comment:
tap -> gesture or pointer interaction?

Reply:
I used "pointer interaction", because a gesture might consist of multiple taps.